src/main/java/com/mateacademy/springmvc/service/UserService.java

```diff
@@ -0,0 +1,17 @@
+package com.mateacademy.springmvc.service;
+
+import com.mateacademy.springmvc.entity.User;
+
+import java.util.List;
+
+public interface UserService {
+    User findByUserName(String userName);
+
+    List<User> getAllUsers();
+
+    void saveUser(User user);
+
+    User getUserById(Long id) throws Throwable;
```

Comment:
Why throws Throwable? Please explain.

Reply:
I added it from previous task where i had some error and i used it) but now i dont need it

Reply:
![image](https://user-images.githubusercontent.com/20267148/58359508-7da50a00-7e8c-11e9-9ab7-9b184634df37.png)